README.md

```diff
@@ -11,7 +11,7 @@
 `regenmaschine` (German for "rain machine") is a simple, clean, well-tested
 Python library for interacting with
 [RainMachine™ smart sprinkler controllers](http://www.rainmachine.com/).
-It gives developers an easy API to manage their controllers over their local
+It gives developers an easy [API](https://rainmachine.docs.apiary.io/) to manage their controllers over their local
```

Comment:
We shouldn't link the API docs to this word, since that implies that that the "API" given to developers by this library _is_ that document. Let's add a new sentence.

Reply:
Moved to lower in the README